.taskcluster.yml

```diff
@@ -266,6 +266,14 @@ tasks:
                                       type: 'directory'
                                       path: '/builds/worker/artifacts'
                                       expires: {$fromNow: '1 year'}
+                                  'public/docker-contexts':
+                                      type: 'directory'
+                                      path: '/builds/worker/checkouts/src/docker-contexts'
```

Comment:
You just changed the `MOZ_FETCHES_DIR` from `checkouts/src` to `fetches` just above, is this correct?

Reply:
Yes, it's unrelated. The `builds/worker/checksouts/src` is where all the artifacts are being checked out. We were initially using that for fetches as well. Which would have been fine as well, until we added `resourcemonitor`. The artifacts produced by it need to stay alive but in a normal taskgraph world, everything that's under `MOZ_FETCHES_DIR` gets wiped-off before tasks run to save space. So removing the artifacts there would've left us with no files after the run-task completed. So instead we're just downloading the artifacts in a different folder (that still gets cleaned-up. We're going to standardize this in the docker image in the near future in https://bugzilla.mozilla.org/show_bug.cgi?id=1654634 anyway so we don't even need to specify this anymore.